Project/Assets/Prairie/Framework/Script/Annotation/ParseAnnotation.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+public class ParseAnnotation {
+
+    private static readonly string IMAGE_TEXT_FULL = "<\\s*img\\s*=\\s*\"?.*?\"?\\s*>";
+    private static readonly string IMAGE_TEXT_SPLIT = "(<\\s*img\\s*=\\s*\"?)|(\"?\\s*>)";
+
+    /// <summary>
+    /// Takes a string and parses text so that textOut gets populated by strings, separated by image paths that are stored in imgOut
+    /// </summary>
+    /// <param name="text"></param>
+    /// <param name="textOut"></param>
+    /// <param name="imgOut"></param>
+    public static void ParseAnnotationText(string text, List<string> textOut, List<string> imgOut)
```

Comment:
The structure would also exist in the contexts of its use inside `AnnotationInteraction`.

I would argue a single `AnnotationContent` structure would be simpler to use inside of `AnnotationInteraction` than coordinating two separate lists (which, at the surface, are unrelated). This structure would also be more open to a possible extension to annotation content in the future.